frontend/src/components/ContentBox.tsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import { Box, Container } from '@mui/material';
 
+import { contentHeaderHeight } from 'src/components/ContentHeader';
+import { topBarHeight } from 'src/features/frame/components/topbar/TopBar';
+
 interface Props {
   maxWidth?: false | 'xs' | 'sm' | 'md' | 'lg' | 'xl';
   children: React.ReactNode;
   noMinPaddingX?: boolean;
 }
 
+/**
+ * A generic top-level container allowing to have the same layout on all
+ * pages.
```

Comment:
Yes, I thought about it and I currently don't remember why I chose to keep the current implementation. Maybe it was to reduce the diff on this PR, so that the changes are focused on the footer itself.

Your suggestion looks good, I think we can do it at the same time we rename the component `ContentBox` -> `PageContent`.